app.drey.Warp.yaml

```
@@ -0,0 +1,52 @@
+app-id: app.drey.Warp
+runtime: org.gnome.Platform
+runtime-version: 42
```

```suggestion
runtime-version: '42'
```

it's yaml, you have to tell it is a string.